openapi.yaml

```diff
@@ -1,279 +1,662 @@
+          description: "The radius of the circle centred on the specified location. Default: 2"
+          required: false
+          schema:
+            type: number
+            format: float
+        - in: query
+          name: filter[distanceUnit]
+          description: "Distance can be specified in following units:
+            Mile - mi or miles (default);
+            Yard - yd or yards;
+            Feet - ft or feet;
+            Inch - in or inch;
+            Kilometer - km or kilometers;
+            Meter - m or meters;
+            Centimeter - cm or centimeters;
+            Millimeter - mm or millimeters;
+            Nautical mile - NM, nmi or nauticalmiles
+          "
+          required: false
+          schema:
+            type: string
+            enum: 
+              - mi
+              - miles
+              - yd
+              - yards
+              - ft
+              - feet
+              - in
+              - inch
+              - km
+              - kilometers
+              - m
+              - meters
+              - cm
+              - centimeters
+              - mm
+              - millimeters
+              - nmi
+              - NM
+              - nauticalmiles
+        - in: query
+          name: filter[isOpen]
+          description: "Set to true to only search for open restaurants/buildings. Default: false to search for all the results"
+          required: false
+          schema:
+            type: boolean
+        - in: query
+          name: filter[campus][oneOf]
+          description: "Name of the campus. Valid options are: corvallis, extension, cascades, hmsc, and other."
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - corvallis
+                - extension
+                - cascades
+                - hmsc
+                - other
+        - in: query
+          name: filter[type]
+          description: "Type of location. Valid options are: building, dining, cultural-center, parking, and other."
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - building
+                - dining
+                - cultural-center
+                - parking
+                - other
+        - in: query
+          name: filter[giRestroom]
+          description: >
+            If true, locations without gender inclusive restrooms will be filtered out.
+          required: false
+          schema:
+            type: boolean
+        - in: query
+          name: filter[parkingZoneGroup][oneOf]
+          description: >
+            Locations in results should match the parkingZoneGroup
+            in this query parameter.
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+              # what are the possible values for this?
```

Comment:
I wasn't sure what the options for this were, and figured it would be better to just ask someone who has access to the source data instead of trying to guess based on online OSU resources

Reply:
<img width="387" alt="Screen Shot 2020-04-16 at 4 38 02 PM" src="https://user-images.githubusercontent.com/9169491/79516595-b3049a00-8000-11ea-8757-bd78650b380b.png">

Reply:
I sent you and @kalcicd and link via Slack for the source data. Let me know if you can access it or now.